test/integration/backward_compatible/rest_value/RestValueTest.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { expect } = require('chai');
+const fs = require('fs');
+const http = require('http');
+const querystring = require('querystring');
+
+const RC = require('../../RC');
+const { Client } = require('../../../../');
+const { deferredPromise } = require('../../../../lib/util/Util');
+
+describe('RestValueTest', function () {
+
+    let cluster;
+    let client;
+    let member;
+
+    before(async function () {
+        cluster = await RC.createCluster(null, fs.readFileSync(__dirname + '/hazelcast_rest.xml', 'utf8'));
+        member = await RC.startMember(cluster.id);
+        client = await Client.newHazelcastClient({ clusterName: cluster.id });
+    });
+
+    after(async function () {
+        await client.shutdown();
+        return RC.terminateCluster(cluster.id);
+    });
+
+    it('client should receive REST events from server as RestValue', async function () {
```

Comment:
Can you explain why you changed here and the github actions issue

Reply:
done was getting called more than once. Once for the callback of the itemListener and once for the socket error. If you check the code, what we are checking is unchanged

Reply:
What happens when it is called multiple times and why it was "ok" before github actions?

Reply:
Calling done() multiple times throw an error according to mocha docs. But still, why this is related to gh actions?

Reply:
To be honest, I have no idea why it fails randomly on Github actions :) I tried to reproduce it locally many times with the exact  Node.js, client, and server versions but couldn't succeed. So, I changed the test a little bit. But if you read it, we are testing the same thing